addons/web_editor/static/src/js/editor/odoo-editor/test/spec/tabs.test.js

```diff
@@ -8,10 +8,12 @@ import {
 } from '../utils.js';
 
 describe('Tabs', () => {
-    const oeTab =  (size = '40px', contenteditable = true) => {
-        return `<span class="oe-tabs"${contenteditable ? '' : ' contenteditable="false"'} style="width: ${size};">\u0009</span>\u200B`
+    const oeTab =  (size = '40px', contenteditable = true, sizeFirst = false) => {
```

Comment:
Do we know the cases that have it with size first ? I mean does it look random or do we have an idea of what causes it ?

Reply:
Could be many things. Would you want me to investigate?

Reply:
Well... I already wasn't a fan of the `oeTab` util before, but your PR makes it even worse... ^^'

Reply:
So... I investigate? I remove the function?